src/simplelink.rs

```diff
@@ -170,4 +180,55 @@ impl SimpleLink {
         try_wlan!(sl_WlanRxFilterSet(op as u8, buf_ptr, buf_size));
         Ok(())
     }
+
+    // File System
+    //
+
```

Comment:
I'd prefer a more idiomatic wrapper than this one. While we won't be able to have the exact same API, something closer to https://doc.rust-lang.org/std/fs/struct.File.html would be nice.